tests/dlpnocc-2/input.dat

```diff
@@ -0,0 +1,53 @@
+#! comparison of DF-CCSD(T) and DLPNO-CCSD(T) without frozen core
+#! Also a test of very_tight parameters
+#! Methane geometry from HTBH.py in databases
+#! The reference DF-CCSD(T) values are stored and not rerun
+
+memory 8 GB
+
+ref_scf                  =    -40.213627016015              #TEST
+
+ref_dfccsd_corl          =     -0.240124468398              #TEST
+ref_dfccsd_tot           =    -40.453751484412              #TEST
+ref_dfccsd_t_corl        =     -0.246845986540              #TEST
+ref_dfccsd_t_tot         =    -40.460473002555              #TEST
+
+ref_dlpnoccsd_corl       =     -0.240195457394              #TEST
+ref_dlpnoccsd_tot        =    -40.453822473402              #TEST
+ref_dlpnoccsd_t_corl     =     -0.246909714224              #TEST
+ref_dlpnoccsd_t_tot      =    -40.460536730232              #TEST
+
+molecule ch4 {
+0 1
+C        0.00000000     0.00000000     0.00000000
+H        0.00000000     1.08744517     0.00000000
+H       -0.51262657    -0.36248173     0.88789526
+H       -0.51262657    -0.36248173    -0.88789526
+H        1.02525314    -0.36248173     0.00000000
+units angstrom
+}
+
+set basis aug-cc-pvtz
+set freeze_core False
+set scf_type df
+set mp2_type df
+set cc_type df
+set nat_orbs False
+set pno_convergence very_tight
+set e_convergence 1.0e-8
+set r_convergence 1.0e-8
+
+print('   Testing DLPNO-CCSD(T) w/o frozen core...')
+val = energy('dlpno-ccsd(t)')
+
```

Comment:
Good test against df-cc, but it'd also be good to compare to more digits against stored dlpno-cc values, so we can know if anything changes in the code.